tests/other_test.go

```diff
@@ -48,3 +52,29 @@ func TestRunVSIPatternWithHPCS(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunOverrideExample(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsQuickStartPattern(t, "slz-ex", overrideExampleTerraformDir)
+	options.SkipTestTearDown = true
+	output, err := options.RunTestConsistency()
+
+	if assert.Nil(t, err, "This should not have errored") &&
+		assert.NotNil(t, output, "Expected some output") &&
+		assert.NotNil(t, options.LastTestTerraformOutputs, "Expected some Terraform outputs") {
+
+		// TERRATEST uses its own internal logger.
+		// The "show" command will produce a very large JSON to stdout which is printed by the logger.
+		// We are temporarily turning the terratest logger OFF (discard) while running "show" to prevent large JSON stdout.
+		options.TerraformOptions.Logger = logger.Discard
+		planStruct, planErr := terraform.InitAndPlanAndShowWithStructE(options.Testing, options.TerraformOptions)
```

Comment:
RunTestConsistency() will already run an Apply and then follow it up with a "InitAndPlan" to verify idempotent. I'm not sure what a second "InitAndPlan" will do here?